packages/ui5-application-writer/src/i18n.ts

```diff
@@ -0,0 +1,34 @@
+import i18next, { TOptions } from 'i18next';
+import translations from './translations/ui5-application-writer.i18n.json';
+
+const NS = 'ui5-application-writer';
+
+/**
+ * Initialize i18next with the translations for this module.
+ */
+export async function initI18n(): Promise<void> {
+    await i18next.init({
+        resources: {
+            en: {
+                [NS]: translations
+            }
+        },
+        lng: 'en',
+        fallbackLng: 'en',
+        defaultNS: NS,
+        ns: [NS]
+    });
+}
+
+/**
+ * Helper function facading the call call to i18next.
+ *
+ * @param key i18n key
+ * @param options additional options
+ * @returns {string} localized string stored for the given key
+ */
+export function t(key: string, options?: TOptions): string {
+    return i18next.t(key, options);
+}
```

Comment:
Can we directly use the `i18next.t()` here instead of wrapping it?

Reply:
I believe it's done like this due ordering of `i18next.init` on load of the file and we don't want consumers to have to manage that process. Also this is replicated code across many modules and should be consolidated to a single i18n utility in another TBI and PR so I will leave it as is for now.